Tests/OutputFilterTest.php

```diff
@@ -18,11 +18,9 @@
 class FilterTestObject
 {
     public $string1;
-
     public $string2;
-
     public $string3;
-
+    public $string4;   
```

Comment:
This seem to be wrong it should be $_string4

Reply:
Good catch, will fix.